src/promptflow-tools/tests/test_handle_openai_error.py

```diff
@@ -301,3 +301,16 @@ def test_aoai_invalid_max_tokens(
             )
         assert error_message in exc_info.value.message
         assert exc_info.value.error_codes == error_codes.split("/")
+
+    def test_aoai_with_vision_model(self, azure_open_ai_connection, mocker: MockerFixture):
```

Comment:
The first test is pass `stop` `None` and `logit_bias` `{}` working with vision model, the test should pass now. This can show the issue https://github.com/microsoft/promptflow/issues/1683 is fixed.

The second test is to pass response_format to trigger "extra fields not permitted" error, looks no need to mock exception.